Sources/TuistAnalytics/CommandEvent.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+public struct CommandEvent: Codable {
+    let name: String
+    let subcommand: String?
+    let duration: TimeInterval
+
+    // SessionEnv
+    let clientId: String
+    let tuistVersion: String
+    let swiftVersion: String
+    let macOSVersion: String
+}
```

Comment:
With Apple brining a new processor architecture, I think we should also track that?
```suggestion
    let processorArchitecture: String  
}
```

or? 

```suggestion
    let processorArchitecture: ProcessorArchitecture
}

enum ProcessorArchitecture {
case x86, appleSilicon
}
```

Reply:
Done